src/plugins/discover/public/application/main/components/layout/discover_main_content.test.tsx

```diff
@@ -26,95 +25,31 @@ import { FetchStatus } from '../../../types';
-import { LocalStorageMock } from '../../../../__mocks__/local_storage_mock';
-import {
-  UnifiedHistogramChartData,
-  UnifiedHistogramLayout,
-} from '@kbn/unified-histogram-plugin/public';
-import { HISTOGRAM_HEIGHT_KEY } from './use_discover_histogram';
-import type { SearchResponse } from '@elastic/elasticsearch/lib/api/types';
-
-jest.mock('@kbn/unified-histogram-plugin/public', () => {
-  const originalModule = jest.requireActual('@kbn/unified-histogram-plugin/public');
-
-  const chartData = {
-    xAxisOrderedValues: [
-      1623880800000, 1623967200000, 1624053600000, 1624140000000, 1624226400000, 1624312800000,
-      1624399200000, 1624485600000, 1624572000000, 1624658400000, 1624744800000, 1624831200000,
-      1624917600000, 1625004000000, 1625090400000,
-    ],
-    xAxisFormat: { id: 'date', params: { pattern: 'YYYY-MM-DD' } },
-    xAxisLabel: 'order_date per day',
-    yAxisFormat: { id: 'number' },
-    ordered: {
-      date: true,
-      interval: {
-        asMilliseconds: jest.fn(),
-      },
-      intervalESUnit: 'd',
-      intervalESValue: 1,
-      min: '2021-03-18T08:28:56.411Z',
-      max: '2021-07-01T07:28:56.411Z',
-    },
-    yAxisLabel: 'Count',
-    values: [
-      { x: 1623880800000, y: 134 },
-      { x: 1623967200000, y: 152 },
-      { x: 1624053600000, y: 141 },
-      { x: 1624140000000, y: 138 },
-      { x: 1624226400000, y: 142 },
-      { x: 1624312800000, y: 157 },
-      { x: 1624399200000, y: 149 },
-      { x: 1624485600000, y: 146 },
-      { x: 1624572000000, y: 170 },
-      { x: 1624658400000, y: 137 },
-      { x: 1624744800000, y: 150 },
-      { x: 1624831200000, y: 144 },
-      { x: 1624917600000, y: 147 },
-      { x: 1625004000000, y: 137 },
-      { x: 1625090400000, y: 66 },
-    ],
-  } as unknown as UnifiedHistogramChartData;
-
-  return {
-    ...originalModule,
-    buildChartData: jest.fn().mockImplementation(() => ({
-      chartData,
-      bucketInterval: {
-        scaled: true,
-        description: 'test',
-        scale: 2,
-      },
-    })),
-  };
-});
+import { searchSourceInstanceMock } from '@kbn/data-plugin/common/search/search_source/mocks';
+import { DiscoverDocuments } from './discover_documents';
+import { FieldStatisticsTab } from '../field_stats_table';
 
 const mountComponent = async ({
   isPlainRecord = false,
-  hideChart = false,
-  isTimeBased = true,
-  storage,
-  savedSearch = savedSearchMock,
-  resetSavedSearch = jest.fn(),
+  viewMode = VIEW_MODE.DOCUMENT_LEVEL,
 }: {
   isPlainRecord?: boolean;
-  hideChart?: boolean;
-  isTimeBased?: boolean;
-  storage?: Storage;
-  savedSearch?: SavedSearch;
-  resetSavedSearch?: () => void;
+  viewMode?: VIEW_MODE;
 } = {}) => {
-  let services = createDiscoverServicesMock();
+  const services = createDiscoverServicesMock();
 
-  if (storage) {
-    services = { ...services, storage };
-  }
+  (services.data.query.queryString.getDefaultQuery as jest.Mock).mockReturnValue({
```

Comment:
We could add it inside `createDiscoverServicesMock`

Reply:
I agree with this, but I tried to update quickly and it broke some unrelated tests that were expecting `getDefaultQuery` to return `undefined`. I think it's a good cleanup task though, just would prefer to save for another PR.